dataactcore/models/domainModels.py

```diff
@@ -24,8 +24,7 @@ class TASLookup(Base) :
       TASLookup.ending_period_of_availability,
       TASLookup.availability_type_code,
       TASLookup.main_account_code,
-      TASLookup.sub_account_code,
-      unique=True)
+      TASLookup.sub_account_code)
```

Comment:
In many of our other domain values tables, we use unique indexes as a safety net for keeping the data clean. 

It makes sense to keep this non-unique index on the tas components; do you think it's necessary to have an second, unique index on the tas + the internal date end field? Or does a combination of carefully written load code + tests suffice to keep things clean?

This is a broader question for data pipelines in general--does everyone enforce data integrity these days?